prime-router/src/test/kotlin/fhirengine/translation/hl7/FhirTransformerTests.kt

```diff
@@ -248,6 +250,34 @@ class FhirTransformerTests {
         assertThat(newValue[0].primitiveValue()).isEqualTo("First Last")
     }
 
+    @Test
+    fun `test transform schema error`() {
+        val bundle = Bundle()
+        bundle.id = "abc123"
+        val resource = Patient()
+        resource.id = "def456"
+        bundle.addEntry().resource = resource
+
+        val elemA = FhirTransformSchemaElement(
+            "elementA",
+            value = listOf("'2.9'"),
+            bundleProperty = "Bundle.entry.resource.ofType(Patient)" +
+                ".extension(\"https://reportstream.cdc.gov/fhir/StructureDefinition/ethnic-group\")" +
+                ".value.coding.version"
+        )
+
+        val schema = FhirTransformSchema(elements = mutableListOf(elemA))
+        schema.name = "ErrorSchema"
+
+        val ex = assertFailsWith<ConfigSchemaElementProcessingException> {
+            FhirTransformer(schema).transform(bundle)
+        }
+        assertThat(ex.message).isEqualTo(
```

Comment:
This is how the exception that occurred in PROD the other day would now be logged